### ISSUE TITLE
Will the code of TransVG++ be released？ thank you

### ISSUE THREAD

**Issue by linhuixiao** [djiajunustc/TransVG#38]
Will the code of TransVG++ be released？
Besides, in TransVG++, the abstract mentioned it‘s a pure-Transformer framework, while in 4.2 Deatails of the paper, it says that "use Mask R-CNN [84] in TransVG++", is it a conflict?
thank you!

**Comment by linhuixiao** [comment]
As for "use Mask R-CNN [84] in TransVG++", is it a writing mistake?

**Comment by linhuixiao** [comment]
As for "use Mask R-CNN [84] in TransVG++", is it a writing mistake?

**Comment by djiajunustc** [comment]
No, it is not a writing mistake.
We use Mask R-CNN to pre-train the ViT backbone parameters in the journal version. You could also regard the pretraining process as training a ViTDet but without the window attention strategy.
